Web/Scotty/Action.hs

```diff
@@ -354,6 +363,19 @@ jsonData = do
         A.Error err -> throwIO $ FailedToParseJSON b $ T.pack err
         A.Success a -> return a
 
+-- | Parse the request body as @x-www-form-urlencoded@ form data and return it.
+--
+--   The form is parsed using 'urlDecodeAsForm'. If that returns 'Left', the
+--   status is set to 400 and an exception is thrown.
+--
+-- NB : Internally this uses 'body'.
+formData :: (FromForm a, MonadIO m) => ActionT m a
```

Comment:
`formData` could be rewritten in terms of `formParams`, [`Form`](https://hackage.haskell.org/package/http-api-data-0.6/docs/Web-FormUrlEncoded.html#t:Form) and [`fromForm`](https://hackage.haskell.org/package/http-api-data-0.6/docs/Web-FormUrlEncoded.html#v:fromForm).

Which shows that yes, it is basically just support for a new encoding.

I initially added this because it was mentioned in #321 but I don't personally think that this is important functionality...